django/cantusdb_project/main_app/management/commands/add_cantus_ultimus_links.py

```diff
@@ -0,0 +1,63 @@
+from main_app.models import Source
+from django.core.management.base import BaseCommand
+
+# This management command goes through all the chants belonging to the specified manuscript
+# and adds a link to the corresponding page on Cantus Ultimus where a user can view the folio
+# of that manuscript. If the source is available on Cantus Ultimus, this script assumes its
+# ID there is the same as its ID on CantusDB.
+
+# This command only works for sources containing chants (i.e. sources in the CANTUS segment,
+# and not sources in the Bower segment which contain sequence objects as opposed to chant
+# objects).
+
+# run with `python manage.py add_cantus_ultimus_links <source_id>`
+
+
+def update_image_link_for_chant(chant):
+    folio = chant.folio
+    number = chant.c_sequence
+    source = chant.source.id
```

Comment:
There would be no place where this was different than the value in line 36, right? In other words, this will be the same for all chants in a single run of this command...

Reply:
Approved...since I think this value will always be the same, I think a refactor would be clearer, but not critical.

Reply:
Yes, `source` will always be the same. This thought occurred to me (mostly from the point of view of optimizing the function to run more quickly), and I erred on the side of writing a function with fewer parameters - shaving a few seconds off the runtime is not that important for a command that I imagine will be run once every few weeks at most.